modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/MessageHandler.java

```diff
@@ -319,13 +254,6 @@ public int clearReadButUndeliveredMessages() {
         lastPurgedTimestamp = System.currentTimeMillis();
         int messageCount = readButUndeliveredMessages.size();
         readButUndeliveredMessages.clear();
-        for (Slot slot : slotsRead.values()) {
-            if (log.isDebugEnabled()) {
-                log.debug("clear tracking of messages for slot = " + slot);
-            }
-            slot.markMessagesOfSlotAsReturned();
```

Comment:
We'll have to discuss that. At the moment we don't have something similar to slot returned in standalone mode.